### ISSUE TITLE
Unable to use npm, npx, but able to use node

### ISSUE THREAD

**Issue by FlashBlaze** [nushell/nushell#1261]
**Describe the bug**
Node, npm and npx are in path of user as well as system variables. However I'm able to use only node and not npm or npx.

**To Reproduce**
1. `node --version` gives version number 
2. `npx --version` command not found
3. `npm --version` command not found

All three commands work in Powershell, Git Bash, CMD.

**Expected behavior**
`npx` and `npm` should display version numbers

**Screenshots**

User variables:
![image](https://user-images.githubusercontent.com/13968549/72915875-0b602e80-3d67-11ea-815f-ad23e854e737.png)

System Variables:
![image](https://user-images.githubusercontent.com/13968549/72915839-f6839b00-3d66-11ea-9cc9-75863a5d1763.png)

**Configuration (please complete the following information):**
 - OS: Windows 10
 - Version 0.8.0

Thanks

**Comment by jonathandturner** [comment]
I'm hoping to fix this issue this weekend. It's happening because npm is coming from npm.cmd:

```
❯ which npm
─────┬─────────────────────────────────┬─────────
 arg │ path                            │ builtin
─────┼─────────────────────────────────┼─────────
 npm │ C:\Program Files\nodejs\npm.cmd │ No
─────┴─────────────────────────────────┴─────────
/c/WINDOWS/system32 via 🦀 v1.40.0
❯ npm
error: Command not found
- shell:1:3
1 | npm
  | ^^^ command not found
```

The .cmd file is just a batch file:

```
:: Created by npm, please don't edit manually.
@ECHO OFF

SETLOCAL

<snip>
```

What we'd like to do very soon is to fix this by detecting when the application is actually a script file, and then running the shell/cmd/powershell behind the scenes for you, so that it's as seamless as running it directly.

**Comment by FlashBlaze** [comment]
Oh, thanks for the information. Should I close this issue after the fix or right now?

**Comment by jonathandturner** [comment]
Once the fix is in, we'll close this issue to let you know it's available.

**Comment by jonathandturner** [comment]
This is fixed by #1264 and should be part of next week's release.
